app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,33 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>AdoptionCenter</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
+<html lang="en-us">
+  <head>
```

Comment:
so, minor point, but it's a super-good idea to make the first tag in the head `<meta charset='utf-8'>` since when the browser pre-processes the first few bytes to figure out the doc encoding, it will hit this tag rather than get invalid/unexpected bytes and guess incorrectly.  I'll find a link for this later if you remind me.